pkg/state/transaction_handler.go

```diff
@@ -264,6 +264,9 @@ func (h *transactionHandler) performTx(
 	if err := snapshot.Apply(h.sa, tx, validatingUTX); err != nil {
 		return txSnapshot{}, errors.Wrap(err, "failed to apply transaction snapshot")
 	}
+
+	// write updates into the updatesChannel here
```

Comment:
Can you explain why and what changes exactly, please?

Reply:
sorry, the comment was left before I implemented the log, it's irrelevant now